app/controllers/application_controller.rb

```diff
@@ -22,6 +22,9 @@ def render_error_message(task)
     elsif params[:activity] == "claimed"
       message = "The task has already been claimed!"
       request_status = 409
+    elsif params[:activity] == "delivered"
+      message = "You haven't claimed this task, please contact support."
```

Comment:
don't understand what this is doing and why support needs to be contacted if something is delivered?